tests/test_pyproject_pep517.py

```diff
@@ -273,6 +273,7 @@ def test_prepare_metdata_for_build_wheel():
         "Requires-Python": ">=3.7",
         "Provides-Extra": "test",
         "Requires-Dist": 'pytest>=6.0; extra == "test"',
+        "License-File": "licenses/LICENSE",
```

Comment:
I don't think it's supposed to be prefixed with licenses

Reply:
Probably related issue: https://github.com/pypa/setuptools/issues/3596

Reply:
Ahh, this is https://peps.python.org/pep-0639/#license-doc-setuptools-wheel - which hasn't been accepted yet.

Reply:
Like hatchling, I'm now following the current form of the unaccepted pep. Nested files with the same name should be supported now too.